js/packages/store/context.ts

```diff
@@ -153,6 +154,10 @@ export type PersistentOptionsDebug = {
 	enable: boolean
 }
 
+export type PersistentOptionsTor = {
+	enable: boolean
```

Comment:
Really not the good type.
accepted values are string :
- `disabled`
- `required`
- `optional` (an acceptable default if *unset* isn't suitable)
- *unset* (default)